### ISSUE TITLE
[OS-REQUEST] Red Hat Enterprise Linux 

### ISSUE THREAD

**Issue by ktheticdev** [TheDarkBug/uwufetch#245]
**OS info**
- OS name: Red Hat Enterprise Linux
- uwufied OS name: Red Cat Nyanterprise Linuwu
- OS family: RHEL
- `/etc/os-release` content:
```
NAME="Red Hat Enterprise Linux"
VERSION="9.2 (Plow)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="9.2"
PLATFORM_ID="platform:el9"
PRETTY_NAME="Red Hat Enterprise Linux 9.2 (Plow)"
ANSI_COLOR="0;31"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:redhat:enterprise_linux:9::baseos"
HOME_URL="https://www.redhat.com/"
DOCUMENTATION_URL="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 9"
REDHAT_BUGZILLA_PRODUCT_VERSION=9.2
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="9.2"
```

**Comment by TheDarkBug** [comment]
Support for rhel (and other new distributions) will be added after completing [libfetch-rewrite](https://github.com/TheDarkBug/uwufetch/tree/libfetch-rewrite)

**Comment by TheDarkBug** [comment]
I tried to write from scratch a logo for rhel, that's the best I can do: 

![2024-02-14T10:28:28,095654045+01:00](https://github.com/TheDarkBug/uwufetch/assets/41623496/96023ea0-2525-42ea-8d5e-b1bc9eb701a6)

I'm not an artist, so if anyone thinks they can do better, go ahead.

The logo is now implemented in [9417838](https://github.com/TheDarkBug/uwufetch/commit/9417838c91aab6088778089b9a3e8330bca53fbd) and will be available in the arch repos starting from the next release.
